larky/src/main/resources/stdlib/xml/etree/ElementTree.star

```diff
@@ -806,7 +806,15 @@ def _ElementTree(element=None, file=None):
                                     of start/end tags
 
         """
-        return Error("Unsupported write method")
+        # return Error("Unsupported write method")
+        qnames, namespaces = _namespaces(self._root, default_namespace)
+        _serialize_xml(
```

Comment:
If you expand the original code, you'll see that there's several serialization methods, maybe what we should do is uncomment  lines 818 to 820 and change line 820 from `elif method not in _serialize` to `elif method != 'xml'`, then we return an error. *THEN* we invoke `_serialize_xml()`

Reply:
Could you check the latest update, I have changed it to be inner function _write just for tostring to use, which does _xml_serialize only?